watertap/tests/test_notebooks.py

```diff
@@ -0,0 +1,98 @@
+"""
+Test Jupyter notebooks in the docs.
+"""
+# stdlib
+import logging
+from pathlib import Path
+import time
+
+# deps
+import nbformat
```

Comment:
To avoid `ImportError`s at collect/import time, we could use [`pytest.importorskip`](https://docs.pytest.org/en/latest/reference/reference.html#pytest-importorskip-ref):

```py
nbformat = pytest.importorskip("nbformat")
```